Source/Tests/TrackableEntities.EF.5.Tests/FamilyDbContextTests.cs

```diff
@@ -72,6 +72,27 @@ public void Apply_Changes_Should_Mark_Modified_Parent_Properties()
         }
 
         [Fact]
+        public void Apply_Changes_Should_Skip_NotMapped_Properties()
+        {
+            // Arrange
+            var context = TestsHelper.CreateFamilyDbContext(CreateFamilyDbOptions);
+            var child = new Child("Child")
+            { 
+                Nickname1 = "Tony",
+                Nickname2 = "Sneed",
+                TrackingState = TrackingState.Modified,
+                ModifiedProperties = new HashSet<string>(new[] { nameof(Child.Nickname1), nameof(Child.Nickname2) })
+            };                                                                           
```

Comment:
I know. These properties are ignored. Should this feature not be implemented and calling `ApplyChanges` will causes an exception.